composables/transaction/collection/delete.ts

```diff
@@ -0,0 +1,29 @@
+import type { ApiPromise } from '@polkadot/api'
+import type {
+  ActionDeleteCollection,
+  ExecuteTransactionParams,
+} from '@/composables/transaction/types'
+
+import { warningMessage } from '@/utils/notification'
+
+export function execDeleteCollection(
```

Comment:
or move it inside `transactionBurn` composable?

Reply:
updated